lib/asset_sync/config.rb

```diff
@@ -224,28 +224,21 @@ def fog_options
 
     # @api
     def add_local_file_paths(&block)
-      @additional_local_file_paths_procs =
-        additional_local_file_paths_procs + [block]
+      @additional_local_file_paths_procs << block
```

Comment:
I prefer non-mutative operations on array
I know this method will mutate the config object anyway, 
It's hard to change that without breaking the interface